packages/renderer/src/lib/ui/Typeahead.svelte

```diff
@@ -181,17 +182,19 @@ function onWindowClick(e: Event): void {
 </script>
 
 <svelte:window on:click={onWindowClick} />
-
 <div
+  role='cell'
```

Comment:
Not sure about the `cell` role, see documentation

> The cell value of the ARIA role attribute identifies an element as being a cell in a tabular container that does not contain column or row header information. To be supported, the cell must be nested in an element with the role of row.[^1]

[^1]: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/cell_role